src/Marketplace/ProduceDetailsScreen.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { View, Image, StyleSheet } from 'react-native';
+import { Text } from 'react-native-paper';
+
+const styles = StyleSheet.create({
+  image: {
+    width: 205,
+    height: 386
+  }
+})
+
+function ProduceDetailsScreen({ route }) {
+  const { image, name, price } = route.params; // need help figuring out how to get the prop types
```

Comment:
label comments like these as TODO: [thing to do] for future ref. for prop types, we can go with the general field type in airtable like number/string/function/etc